### PR TITLE
Cost Tracking Change Log

### PR DESCRIPTION
**Pull request by marlenc3**

### CODE REVIEW

File: fern/docs/content/change-log/2024-08.mdx [hunk 1]
```diff
@@ -32,3 +32,15 @@ you can view a history of all Evaluation runs and go back to view the results of
 
 This is particularly helpful if you want to do things like compare the results of two different Evaluation runs,
 download the results of a past Evaluation run, or simply view the Test Cases that existed at that time.
+
+## Prompt Cost Tracking
```
Comment (dvargas92495) [inline fern/docs/content/change-log/2024-08.mdx:36]: [blocking] let's add this towards the top, we go from most recent -> least recent

File: fern/docs/content/change-log/2024-08.mdx [hunk 2]
```diff
@@ -32,3 +32,15 @@ you can view a history of all Evaluation runs and go back to view the results of
...
+
+_August 8th, 22024_
+
+You can now see cost tracking in prompt sandboxes, both in the prompt editor and in the comparison tab!
+
+![Prompt Sandbox With Cost Tracking](https://storage.cloud.google.com/vellum-public/help-docs/changelogs/2024-08/prompt_sandbox_with_cost_tracking.png)
+![Prompt Sandbox Comparison With Cost Tracking](https://storage.cloud.google.com/vellum-public/help-docs/changelogs/2024-08/prompt_sandbox_comparison_with_cost_tracking.png)
+
+If you're curious about the pricing information, feel free to additionally look into a Model's page.
+![MLModel Detail Page with Billing Config](mlmodel_detail_page_with_cost_config.png)
```
Comment (dvargas92495) [inline fern/docs/content/change-log/2024-08.mdx:46]: [blocking] CI is breaking on this link

File: fern/docs/content/change-log/2024-08.mdx [hunk 3]
```diff
@@ -32,3 +32,15 @@ you can view a history of all Evaluation runs and go back to view the results of
...
+## Prompt Cost Tracking
+
+_August 8th, 22024_
+
+You can now see cost tracking in prompt sandboxes, both in the prompt editor and in the comparison tab!
+
+![Prompt Sandbox With Cost Tracking](https://storage.cloud.google.com/vellum-public/help-docs/changelogs/2024-08/prompt_sandbox_with_cost_tracking.png)
+![Prompt Sandbox Comparison With Cost Tracking](https://storage.cloud.google.com/vellum-public/help-docs/changelogs/2024-08/prompt_sandbox_comparison_with_cost_tracking.png)
+
+If you're curious about the pricing information, feel free to additionally look into a Model's page.
```
Comment (dvargas92495) [inline fern/docs/content/change-log/2024-08.mdx:45]:
```suggestion
If you're curious about the pricing information, feel free to look into a Model's details page.
```

File: fern/docs/content/change-log/2024-08.mdx [hunk 4]
```diff
@@ -32,3 +32,15 @@ you can view a history of all Evaluation runs and go back to view the results of
 
 This is particularly helpful if you want to do things like compare the results of two different Evaluation runs,
 download the results of a past Evaluation run, or simply view the Test Cases that existed at that time.
+
+## Prompt Cost Tracking
+
+_August 8th, 22024_
+
+You can now see cost tracking in prompt sandboxes, both in the prompt editor and in the comparison tab!
```
Comment (dvargas92495) [inline fern/docs/content/change-log/2024-08.mdx:40]: [nit] Let's add some copy here that says that we are tracking the cost from the provider. Let's also add something towards the end hinting that it's available for OpenAI, Anthropic, and Google models, with the rest to follow in the coming days

**Review by dvargas92495 — CHANGES_REQUESTED** [review]

File: fern/docs/content/change-log/2024-08.mdx [hunk 5]
```diff
@@ -2,6 +2,21 @@
 title: Changelog | August, 2024
 ---
 
+## Prompt Cost Tracking
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-08.mdx:5]:
```suggestion
## Prompt Sandbox Cost Tracking
```

File: fern/docs/content/change-log/2024-08.mdx [hunk 6]
```diff
@@ -2,6 +2,21 @@
...
+
+_August 9th, 22024_
+
+You can now see cost tracking for input and output tokens in prompt sandboxes, both in the prompt editor and in the comparison tab!
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-08.mdx:9]:
```suggestion
You can now see the dollar cost of a Prompt's execution within both a Prompt Sandbox's Prompt Editor and Comparison Mode views.
```

File: fern/docs/content/change-log/2024-08.mdx [hunk 7]
```diff
@@ -2,6 +2,21 @@
...
+Cost of input and output tokens are being tracked from providers.
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-08.mdx:10]: These costs are calculated using model providers' publicly available pricing data in conjunction with the number of input/output tokens used.

File: fern/docs/content/change-log/2024-08.mdx [hunk 8]
```diff
@@ -2,6 +2,21 @@
...
+
+![Prompt Sandbox With Cost Tracking](https://storage.cloud.google.com/vellum-public/help-docs/changelogs/2024-08/prompt_sandbox_with_cost_tracking.png)
+![Prompt Sandbox Comparison With Cost Tracking](https://storage.cloud.google.com/vellum-public/help-docs/changelogs/2024-08/prompt_sandbox_comparison_with_cost_tracking.png)
+
+If you're curious about the pricing information, feel free to look into a Model's details page.
+![MLModel Detail Page with Billing Config](https://storage.cloud.google.com/vellum-public/help-docs/changelogs/2024-08/mlmodel_detail_page_with_cost_config.png)
+
+Most popular models are already supported, with even more models following in the coming days.
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-08.mdx:18]:
```suggestion
If you're curious about a given model's pricing, you can view details in the Model's detail page.
![MLModel Detail Page with Billing Config](https://storage.cloud.google.com/vellum-public/help-docs/changelogs/2024-08/mlmodel_detail_page_with_cost_config.png)

Most popular models already have pricing information populated, with support for even more models following in the coming days.

Showing cost information in Prompt Sandboxes is just the first step! We'll expose cost details throughout more of Vellum over time.
```

File: fern/docs/content/change-log/2024-08.mdx [hunk 9]
```diff
@@ -2,6 +2,21 @@
...
 
+## Prompt Cost Tracking
+
+_August 9th, 22024_
+
+You can now see cost tracking for input and output tokens in prompt sandboxes, both in the prompt editor and in the comparison tab!
+Cost of input and output tokens are being tracked from providers.
+
+![Prompt Sandbox With Cost Tracking](https://storage.cloud.google.com/vellum-public/help-docs/changelogs/2024-08/prompt_sandbox_with_cost_tracking.png)
+![Prompt Sandbox Comparison With Cost Tracking](https://storage.cloud.google.com/vellum-public/help-docs/changelogs/2024-08/prompt_sandbox_comparison_with_cost_tracking.png)
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-08.mdx:13]: Looking at these images, I feel like we should round to 9 significant digits. the long-tail of 99999999 feels overkill.

Comment (noanflaherty) [reply fern/docs/content/change-log/2024-08.mdx:13]: Also, while true, the definition of Vellum here is pretty gruesome and makes me think we could use a screenshot of another example 😅

**Review by noanflaherty — APPROVED** [review]
Nice work! Let's please iterate based on my suggestions before merging, but we can turn on the feature flag for everyone whenever.

In case it's helpful, we have some guidelines for writing stuff for the changelog here: https://www.notion.so/vellum-ai/Guide-to-Writing-Product-Updates-6ebad76d47274d7180a55eef13946bbd

**Review by dvargas92495 — APPROVED** [review]

File: fern/docs/content/change-log/2024-08.mdx [hunk 10]
```diff
@@ -2,6 +2,21 @@
 title: Changelog | August, 2024
 ---
 
+## Prompt Sandbox Cost Tracking
+
+_August 9th, 22024_
```
Comment (dvargas92495) [inline fern/docs/content/change-log/2024-08.mdx:7]:
```suggestion
_August 9th, 2024_
```

**Pull request merged**